config/locales/diaspora/en.yml

```diff
@@ -851,12 +851,15 @@ en:
       message: |-
         Hello!
 
-        You have been invited to join diaspora*!
+        You have been invited to join diaspora* by %{diaspora_id}!
 
         Click this link to get started
 
         [%{invite_url}][1]
 
+        You can add %{diaspora_id} to the pod you might be currently using.
```

Comment:
I think "Or you can add %{diaspora_id} to your contacts if you already have an account." or something like this is clearer. Or maybe @goobertron has a better idea.

`diaspora_id` shouldn't be `nil`, because every user has a diaspora-ID. I don't know what happens if the user has closed his account, and if the invitation still works.